VirtusLab-Open-Source/strapi-plugin-comments
# feat: comments custom field [CU-2tn2923]

## Ticket

https://app.clickup.com/t/2tn2923

**NOTE:** This PR is dependent on https://github.com/VirtusLab-Open-Source/strapi-typed/pull/14

## Summary

What does this PR do/solve?

- Custom field for comments

## Test Plan

- install plugin on Strapi version `4.4.0-alpha.0+`
- use content builder or add manually custom field
- create/edit an entity
- custom field should be visible and editable
